we.go

```diff
@@ -562,33 +568,24 @@ func marshalLang(rv map[string]interface{}, r *resp) error {
 }
 
 func main() {
-	var numdays = 3
-
 	if len(config.APIKey) == 0 {
 		log.Fatal("No API key specified. Setup instructions are in the README.")
 	}
 	params = append(params, "key="+config.APIKey)
 
-	for _, arg := range os.Args[1:] {
-		if v, err := strconv.Atoi(arg); err == nil && len(arg) == 1 {
-			numdays = v
-		} else {
-			config.City = arg
-		}
-	}
+	flag.Parse()
+	config.City = city
 
 	if len(config.City) > 0 {
 		params = append(params, "q="+url.QueryEscape(config.City))
 	}
 	params = append(params, "format=json")
-	params = append(params, "num_of_days="+strconv.Itoa(numdays))
+	params = append(params, "num_of_days="+strconv.Itoa(days))
 	params = append(params, "tp=3")
 	if config.Lang != "" {
 		params = append(params, "lang="+config.Lang)
 	}
 
-	// fmt.Fprintln(os.Stderr, params)
-
 	res, err := http.Get(uri + strings.Join(params, "&"))
```

Comment:
Please don't remove those comments. If you want, add a 'debug' flag to enable them in another pull request.